go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bank-vaults/vault-secrets-webhook
 
-go 1.21.1
+go 1.21
```

Comment:
no need to change this

Reply:
It threw errors about 1.21.1 not being a thing when i built it